crates/secrets/Cargo.toml

```diff
@@ -21,9 +21,11 @@ async-trait.workspace = true
 base64 = "0.22.0"
 keyring = { version = "2.3.2", optional = true }
 secrecy = "0.8.0"
+aws-config = { version = "1.1.10", optional = true}
 aws-sdk-secretsmanager = { version = "1.21.0", optional = true }
+aws-sdk-sts = { version = "1.19.0", optional = true }
```

Comment:
`aws-sdk-sts` - we don't actually bring additional dependency. It is used by `aws-config` already, so we just re-use it to validate AWS access